src/models/RunGraph.ts

```diff
@@ -1,4 +1,6 @@
 import { ColorSource } from 'pixi.js'
+import { DeepRequired } from 'ts-essentials'
+import { RunGraph } from '..'
```

Comment:
This should probably be an aliased import

Reply:
Idk why this was here anyway. Its unused.